File: Java/PE0001.java
public class PE0001 {

  public static void main(String[] args){
    System.out.println(new PE0001().solve());
  }

  public int solve() {
    int sum = 0;
    for(int i = 0; i < 1000; i++){
      if (i % 3 == 0 || i % 5 == 0){
        sum += i;
      }
    }
    return sum;
  }
}

File: Ruby/PE0001.rb
def isMult3or5(num)
  num % 5 == 0 || num % 3 == 0
end


def mult3or5(upto=1000)
  sum = 0
  (1...upto).each{ |num| sum += num if isMult3or5(num) }
  sum
end


puts mult3or5
